Libplanet/Blockchain/BlockChain.cs

```diff
@@ -441,6 +456,11 @@ public bool ContainsBlock(HashDigest<SHA256> blockHash)
         /// <see cref="Transaction{T}"/> with a given <paramref name="txId"/>.</exception>
         public Transaction<T> GetTransaction(TxId txId)
         {
+            if (StagePolicy.Get(this, txId, includeUnstaged: true) is { } tx)
+            {
+                return tx;
+            }
+
             _rwlock.EnterReadLock();
```

Comment:
Are doesn't need _rwlock.ExitWriteLock() in this method?

Reply:
You mean, why `_rwlock` is necessary here? It's for protecting `_transactions` from other threads.

Reply:
I misread about the lock. thanks for your answer.